assets/scripts/store/reducers/__tests__/street.test.js

```diff
@@ -222,6 +222,35 @@ describe('street reducer', () => {
     })
   })
 
+  describe('CLEAR_LOCATION', () => {
+    it('sets street.location to be null and street name to DEFAULT_NAME if userUpdated = False', () => {
+      const existingStreet = {
+        name: 'test street',
+        userUpdated: false,
+        location: {
+          latlng: [0, 0],
+          label: 'test street',
+          hierarchy: {
+            country: 'country',
+            locality: 'locality',
+            neighbourhood: 'neighbourhood',
+            street: 'street'
+          },
+          geometryId: null,
+          intersectionId: null
+        }
+      }
+
+      expect(
+        reducer(Object.assign({}, existingStreet), actions.clearLocation())
```

Comment:
Recently, I realized that `Object.assign({}, whatever` is mostly equivalent to saying `{ ...whatever }`. Because it looks cleaner, I recommend doing that from now on.

Reply:
Should I do this for all the `Object.assign({ }, whatever)` in `street.test.js`, including the older test or should I do it only to the ones related to this PR and fix the others in another branch?